cnames_active.js

```diff
@@ -2764,6 +2764,7 @@ var cnames_active = {
   "yusril": "yusrilhs.github.io", // noCF
   "yuye": "cyea.github.io",
   "yvm": "tophat.github.io/yvm",
+  "zaimu": "zaimu.netlify.app", //noCF
```

Comment:
```suggestion
  "zaimu": "zaimu.netlify.app", // noCF
```
Missing space, consistent formatting